contrib/automove/snapswitch.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Change all Snapcast groups to play current stream
+After=network.target time-sync.target sound.target avahi-daemon.service
```

Comment:
I think you'll need to add "Wants=" with the same arguments as "After=" for the ordering to work.